static/scss/email-composition-dialog.scss

```diff
@@ -165,3 +165,20 @@
     }
   }
 }
+
+.toolbar-below {
+  padding: 6px 5px 0;
+  display: -webkit-box;
```

Comment:
The autoprefixer should take care of this and the `-ms-flexbox` below. You just need `display: flex`